### PR TITLE
Added configuration for Corpus python package

### PR DESCRIPTION
**Pull request by talhajunaidd**

### CODE REVIEW

File: recipes/corpus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - Corpus
+  requires:
+    - nose
+
+about:
+  home: https://github.com/aneeshb413/Corpus
+  license: MIT
```
Comment (grst) [inline recipes/corpus/meta.yaml]: please include a license file

File: recipes/corpus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  license_family: MIT
+  summary: A simple English dictionary for Python
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - talhajunaidd
+    - sadeemahmad
```
Comment (grst) [inline recipes/corpus/meta.yaml]: @sadeemahmad, are you OK being added as maintainer for this conda package?

File: recipes/corpus/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ffa894899350675f874bab9e93ef07326aeb5fc1e9ef62d25729d4b2903c63cc
+
+build:
```
Comment (grst) [inline recipes/corpus/meta.yaml]: Can this be a `noarch:python` package? 
https://conda-forge.org/docs/meta.html#building-noarch-packages

File: recipes/corpus/meta.yaml (hunk 2 above)
Comment (Sadeemahmad) [reply recipes/corpus/meta.yaml]: Yes

**Pull request closed without merging**